jazz_services-handler/components/crud/get.js

```diff
@@ -0,0 +1,34 @@
+const request = require('request');
+
+module.exports = (inputs, callback) => {
+
+	var svcPayload = {
+		uri: inputs.SERVICE_API_URL + inputs.SERVICE_API_RESOURCE + "?domain=" + inputs.DOMAIN + "&service=" + inputs.SERVICE_NAME,
+		method: 'GET',
+		headers: {
+			'Authorization': inputs.TOKEN
+		},
+		rejectUnauthorized: false
+	};
+
+	request(svcPayload, function (error, response, body) {
+		if (response.statusCode === 200 && body && body.data ) {
+			var results = JSON.parse(body);
```

Comment:
if body needs to be parsed then body.data in if condition will give you undefined.